### PR TITLE
Added relational operator printer for various languages

### LINKED ISSUE #17238: Relational printing
```python3
from sympy import *

from sympy.printing.ccode import ccode
from sympy.printing.cxxcode import cxxcode
from sympy.printing.fcode import fcode
from sympy.printing.glsl import glsl_code
from sympy.printing.jscode import jscode
from sympy.printing.julia import julia_code
from sympy.printing.mathematica import mathematica_code
from sympy.printing.octave import octave_code
from sympy.printing.pycode import pycode
from sympy.printing.rcode import rcode
from sympy.printing.rust import rust_code

x = Symbol('x')

print(ccode(Eq(x, 1)))
print(cxxcode(Eq(x, 1)))
print(glsl_code(Eq(x, 1)))
print(fcode(Eq(x, 1)))
print(jscode(Eq(x, 1)))
print(julia_code(Eq(x, 1)))
print(mathematica_code(Eq(x, 1)))
print(octave_code(Eq(x, 1)))
print(pycode(Eq(x, 1)))
print(rcode(Eq(x, 1)))
print(rust_code(Eq(x, 1)))
```
Result
```
x == 1
x == 1
Eq(x, 1)
      x == 1
Eq(x, 1)
Eq(x, 1)
Eq(x, 1)
Eq(x, 1)
(x == 1)
x == 1
Eq(x, 1)
```
glsl, javascript, julia, mathematica, octave, rust code printers are probably printing equality in a wrong way.
They are false-positively looking up for `StrPrinter._print_Relational`

C or Fortran printers are overriding `_print_Relational`, so they are the only things working.

### PR DESCRIPTION
**Pull request by ritesh99rakesh**
#### References to other Issues or PRs

Fixes #17238 

#### Brief description of what is fixed or changed
Added relational operator printer for following languages:
1. glsl
2. js
3. julia
4. mathematica
5. octave
6. rust

I have also verified that they are correct. Added tests for same.
Ping @sylee957 
#### Release Notes

- printing
    - added relational operator printing for various languages

### CODE REVIEW

File: sympy/printing/glsl.py [hunk 1]
```diff
@@ -281,6 +281,12 @@ def _print_int(self, expr):
     def _print_Rational(self, expr):
         return "%s.0/%s.0" % (expr.p, expr.q)
 
+    def _print_Relational(self, expr):
+        lhs_code = self._print(expr.lhs)
+        rhs_code = self._print(expr.rhs)
+        op = expr.rel_op
+        return ("{0} {1} {2}").format(lhs_code, op, rhs_code)
```
Comment (sylee957) [inline sympy/printing/glsl.py]:
```suggestion
        return "{0} {1} {2}".format(lhs_code, op, rhs_code)
```
Can you remove these additional parentheses here and every below?

**Review by sylee957 — COMMENTED** [review]
Can you also add _print_not_supported in `CodePrinter`
```_print_Relational = _print_not_supported```

**Comment by ritesh99rakesh** [comment]
@sylee957 done. Please review.

File: sympy/printing/fcode.py [hunk 2]
```diff
@@ -365,6 +365,12 @@ def _print_Float(self, expr):
             return "%sd%s" % (printed[:e], printed[e + 1:])
         return "%sd0" % printed
 
+    def _print_Relational(self, expr):
+        lhs_code = self._print(expr.lhs)
+        rhs_code = self._print(expr.rhs)
+        op = expr.rel_op
+        return "{0} {1} {2}".format(lhs_code, op, rhs_code)
```
Comment (sylee957) [inline sympy/printing/fcode.py]: For fortran case, I think that it can be better to merge `_print_Equality` and `_print_Unequality` here.

Comment (ritesh99rakesh) [reply sympy/printing/fcode.py]: I have removed `_print_Equality` and `_print_Unequality`. The tests already exist under `test_fcode`  as `test_fcode_Relational()`.
Please review.

**Comment by sylee957** [comment]
Looks good

**Pull request merged**